Sources/WalletConnectSign/Auth/Services/App/SessionAuthRequestService.swift

```diff
@@ -29,7 +29,7 @@ actor SessionAuthRequestService {
         var params = params
         let pubKey = try kms.createX25519KeyPair()
         let responseTopic = pubKey.rawRepresentation.sha256().toHexString()
-        let protocolMethod = SessionAuthenticatedProtocolMethod(ttl: params.ttl)
+        let protocolMethod = SessionAuthenticatedProtocolMethod.responseApprove(ttl: params.ttl)
```

Comment:
here we have a request not response, should not have an effect on the behaviour but this init reads strangely.

Reply:
I agree, and I started out by trying to change [`method`](https://github.com/WalletConnect/WalletConnectSwiftV2/blob/3afcf07d5af4a391104ba469f55f6304496933e2/Sources/WalletConnectNetworking/ProtocolMethod.swift#L4) into a `static var` however other parts of the codebase were depending on this being an instance `var`.